CHANGELOG.md

```diff
@@ -9,9 +9,15 @@ A wrapper is available in the `@apostrophecms/widget-type` module to register op
 For example, calling `self.addWidgetOperation` in the ``@apostrophecms/image-widget`` module will apply the operation exclusively to image widgets.
 A `secondaryLevel: true` option is available to add operations to the widget's controls context menu.
 
+### Changes
+
+* Enable widget live preview by default.
+
 ### Fixes
 
 * Fixes all eslint warnings.
+* Fixes a RT bug where including `table` in `toolbar` but omitting an `insert` array crashed the rich text editor. 
```

Comment:
```suggestion
* Fixes a rich-text bug where including `table` in `toolbar` but omitting an `insert` array crashed the rich text editor. 
```

Reply:
This is not a change made by this PR, I'm not sure how it ended here.

Reply:
This entry doesn't belong here and is removed